lib/MusicBrainz/Server/Controller/Role/Load.pm

```diff
@@ -89,6 +90,13 @@ role
                 my %opts;
 
                 if (is_positive_integer($link_type_id)) {
+                    my $link_type = $c->model('LinkType')->get_by_id($link_type_id);
+                    unless ($link_type) {
+                        $c->stash(
+                            message  => l('The provided relationship type does not exist'),
```

Comment:
We have the following string in root/components/RelationshipsTable.js which makes it sound like this case should be handled already: `The provided relationship type ID is not valid.`. Erroring seems fine, but perhaps we can use the existing string and remove that unused fallback.

Reply:
Oh yeah, I guess that fallback doesn't actually work because it runs if there's no paged group returned, but there is - just a useless one. I actually even tried to find an existing string to reuse and failed, sigh. Thanks!